salem/sio.py

```diff
@@ -126,22 +126,25 @@ def read_shapefile_to_grid(fpath, grid):
 
 def _wrf_grid_from_dataset(ds):
     """Get the WRF projection out of the file."""
-
-    pargs = dict()
+    pargs = {}
     if hasattr(ds, 'PROJ_ENVI_STRING'):
         # HAR and other TU Berlin files
-        dx = ds.GRID_DX
-        dy = ds.GRID_DY
-        pargs['lat_1'] = ds.PROJ_STANDARD_PAR1
-        pargs['lat_2'] = ds.PROJ_STANDARD_PAR2
-        pargs['lat_0'] = ds.PROJ_CENTRAL_LAT
-        pargs['lon_0'] = ds.PROJ_CENTRAL_LON
-        pargs['center_lon'] = ds.PROJ_CENTRAL_LON
+        dx = ds.GRID_DX or ds.DX
+        dy = ds.GRID_DY or ds.DY
```

Comment:
This doesnt look like a good idea because if GRID_DX is not available it will raise an error, not go to ds.DX